proxy/src/http/source.rs

```diff
@@ -274,16 +276,28 @@ mod handler {
 
     /// Fetch a [`coco::Blob`].
     pub async fn blob(
-        peer: Arc<Mutex<coco::PeerApi>>,
+        api: Arc<Mutex<coco::PeerApi>>,
```

Comment:
Was this @xla changing the naming? xD

Reply:
Indeed, he was : D

Reply:
Too many peers in the house. May th real peer please stand up!